python/torch_mlir_e2e_test/test_suite/reduction.py

```diff
@@ -462,6 +462,60 @@ def ReduceMaxUnsignedIntModule_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class ReduceAmaxSingleDim(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, a):
+        return torch.ops.aten.amax(a, 1)
+
+@register_test_case(module_factory=lambda: ReduceAmaxSingleDim())
+def ReduceAmaxSingleDim_basic(module, tu: TestUtils):
+    module.forward(tu.rand(3, 4, 5, high=100))
+
+# ==============================================================================
+
+class ReduceAmaxMultiDim(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, a):
+        return torch.ops.aten.amax(a, (0,2))
+
+@register_test_case(module_factory=lambda: ReduceAmaxMultiDim())
+def ReduceAmaxMultiDim_basic(module, tu: TestUtils):
+    module.forward(tu.rand(3, 4, 5, high=100))
+
+# ==============================================================================
+
+class ReduceAmaxKeepdim(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, a):
+        return torch.ops.aten.amax(a, (0,2), True)
```

Comment:
nit: space afer comma, and use `keepdim=True` to make it clear what `True` corresponds to